app/assets/v2/js/cart.js

```diff
@@ -375,6 +374,12 @@ Vue.component('grants-cart', {
       return isPolygon ? 'MATIC' : 'ETH';
     },
 
+    gitcoinAddress() {
+      let isPolygon = this.networkId === '80001' || this.networkId === '137';
```

Comment:
```suggestion
      let isPolygon = networkId === '80001' || networkId === '137';
```

Reply:
@gdixon : I do not think this will work. You want to use the data attribute `networkId` of the vue component?